theano/misc/check_blas.py

```diff
@@ -240,26 +246,88 @@ def test():
         GT 220                                                             3.80s
         GT 210                                                      6.35s
         8500 GT                                                                   10.68s
+
+        Results for larger matrices.
+        There were 10 executions of gemm in float32
+        with matrices of shape 5000x5000 (M=N=K=5000).
+        All memory layout was in C order.
+
+        cuda version      6.5    6.0    5.5    5.0    4.2    4.1    4.0    3.2    3.0   # note
```

Comment:
Add only the columns 7.5 (annonced), 7.0, and 6.5.

I think it isn't useful to keep to old version now and we have the new.